apps/microsoft-teams/frontend/test/mocks/mockChannels.ts

```diff
@@ -24,7 +24,7 @@ const mockChannels: TeamsChannel[] = [
   },
   {
     id: '19:3bccfda604454e63bd839399e6752ba3@thread.tacv2',
-    name: 'General',
+    name: 'Sales General',
```

Comment:
Add Sales to make each item of this mock unique.  This helps when using react testing lib `getBy...`

Reply:
Makes sense! However there is a reason we have this named `General`. In Teams, each Team must have at least 1 channel, and that channel's name is always `General`. So in this case the `Sales Team` Team needs to have a channel named `General`. I know this is only a mock, but just wanted to explain why we have it this way!

Reply:
Gotcha, that makes sense.  With that in mind, I reverted the changes to this file and updated the unit test that needed it to use `teamName` instead of name.

Thanks, good callout.